### ISSUE TITLE
OAuth example does not work

### ISSUE THREAD

**Issue by ndjdev** [eversign/eversign-python-sdk#2]
Trying out the OAuth code and the example https://github.com/eversign/eversign-python-sdk/blob/master/examples/oauth_token.py doesn't work:

```python
    return esclient.create_document(document)
  File "/Volumes/Users/nick/.virtualenvs/env/lib/python3.6/site-packages/eversign/client.py", line 130, in create_document
    return_type=Document)
  File "/Volumes/Users/nick/.virtualenvs/env/lib/python3.6/site-packages/eversign/client.py", line 500, in _request
    raise EversignException(code, _type, info)
eversign.utils.EversignException: (101, 'missing_access_key', 'You have not supplied an API Access Key. [Required format: access_key=YOUR_ACCESS_KEY]')
```
This works and connects if you use the non-oauth connection code, but I don't get any callbacks to the app's callback URL.

Thanks

**Comment by clmns** [comment]
Hi Nick,
did you try to run the oauth.py file before oauth_example.py? The authentication and callback happens there, then you can put the code and state into the config.py file.

Thanks!

**Comment by ndjdev** [comment]
Yes I did, I had my code and state set up properly. "missing_access_key" is coming from the API rather than the SDK, however I've checked the headers and "Bearer: XXX" is being set correctly.

**Comment by ndjdev** [comment]
So I've done a bit more testing and it appears as though https://api.eversign.com/api/business gets called fine but then https://api.eversign.com/api/document creates the "missing_access_key" error. Should it be working that way?

**Comment by dominikkukacka** [comment]
Hi,
just to clarify this is the flow you have to do:

* run `python examples/oauth.py` to generate the URL (https://eversign.com/oauth/authorize?client_id=xxxxxxxxxxx&state=test)
* redirect the user to the URL, which should look like that:
![image](https://user-images.githubusercontent.com/243319/46147565-159f1500-c266-11e8-8a7d-1398a49628f9.png)
* he clicks on *authorize*
* he will be redirected to something like: `http://localhost/example/oauth/callback.php?code=XXXXXXXXXX&state=test`
* you extract the `code`and place it in the config
* you run `python examples/oauth_token.py`
* it returns the business id of the default business but you can do any call you like there.

I hope this clarifies it.
Best,
Dominik

**Comment by ndjdev** [comment]
Yep done all that, getting the business info works fine (confirming setup is correct) but trying to POST to /document does not.

**Comment by dominikkukacka** [comment]
Oh, you are right. We found the issue and will release a fix as fast as possible.

**Comment by ndjdev** [comment]
Amazing thanks

**Comment by dominikkukacka** [comment]
we just released 1.1 (https://pypi.org/project/eversign/1.1/) which fixes this bug.
